.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Continuous Integration'
+on: [ push, pull_request ]
+
+env:
+  OSSRH_USERNAME: ${{ secrets.SONATYPE_S01_USERNAME }}
+  OSSRH_TOKEN: ${{ secrets.SONATYPE_S01_PASSWORD }}
+  MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_CENTRAL_GPG_PASSWORD }}
+jobs:
+  BuildAndPublish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v3
+        with: # running setup-java again overwrites the settings.xml
+          java-version: 8
+          distribution: 'temurin'
+          cache: 'maven'
+          server-id: ossrh # Value of the distributionManagement/repository/id field of the pom.xml
+          server-username: OSSRH_USERNAME # env variable for username in deploy
+          server-password: OSSRH_TOKEN # env variable for token in deploy
+          gpg-private-key: ${{ secrets.MAVEN_CENTRAL_GPG_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE # env variable for GPG private key passphrase
+      - name: Publish alfresco-value-assistance-repo packages
+        if: ${{ startsWith(github.ref, 'refs/heads/master') }}
+        run: mvn --batch-mode deploy -Dgpg.skip=true
+        working-directory: alfresco-value-assistance-repo
+      - name: Publish alfresco-value-assistance-share packages
+        if: ${{ startsWith(github.ref, 'refs/heads/master') }}
+        run: mvn --batch-mode deploy -Dgpg.skip=true
```

Comment:
Do we use the GPG later? This is for signing the package? why is it being skipped

Reply:
There where some errors with the signing of the artifacts.
`Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.5:sign (sign-artifacts) on project alfresco-value-assistance-repo: Exit code: 2`

Possible other solution to try out:
https://github.com/diva-e/parallel-test-runner/commit/e198e6e26188bca9949a3880d153fb8ceb1ff988

Reply:
solved the issue by upgrading the gpg signing plugin.